### PR TITLE
Update OSG container actions to automatically generate the TIMESTAMP_TAG build variable [SOFTWARE-5317]

### PR DESCRIPTION
**Pull request by AidenCohen31**

### CODE REVIEW

**Review by brianhlin — CHANGES_REQUESTED** [review]
Note that the timestamp is just the last bit of the `TIMESTAMP_TAG` (see the description for an example https://opensciencegrid.atlassian.net/browse/SOFTWARE-5317)

Also, in the future, make sure to squash/reword commits before submitting the PR! You may not want to rebase in this case since it appears that a merge commit snuck in

**Comment by AidenCohen31** [comment]
squashed commits

**Comment by AidenCohen31** [comment]
added more changes [SOFTWARE 5319]

**Comment by brianhlin** [comment]
@AidenCohen31 it looks like there are some merge conflicts, could you sort those out?

**Comment by AidenCohen31** [comment]
Ok

**Comment by AidenCohen31** [comment]
needed to add the "" to the if statement as it would throw an error if name not specified

File: action.yml [hunk 1]
```diff
@@ -25,7 +31,10 @@ inputs:
     required: false
     default: ''
     
-
+outputs:
+      tags: 
+        description: generated timestamp
```
Comment (brianhlin) [inline action.yml]:
```suggestion
        description: generated timestamp tag
```

File: action.yml [hunk 2]
```diff
@@ -34,7 +43,8 @@ runs:
   - id: generate-image-name
     shell: bash
     run: |
-      if [[ -n ${{ inputs.image_name }} ]]; then
+
```
Comment (brianhlin) [inline action.yml]:
```suggestion
```

Get rid of extra newline

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**